src/library/Uno.Material/Styles/Controls/v2/CheckBox.xaml

```diff
@@ -558,7 +565,7 @@
 								 Stretch="Uniform"
 								 Width="{ThemeResource CheckBoxFocusAreaSize}"
 								 Height="{ThemeResource CheckBoxFocusAreaSize}"
-								 Fill="{ThemeResource CheckBoxStateCircleFillChecked}"
+								 Fill="Transparent"
```

Comment:
As we are updating this value, be careful some visual states do not have the Fill property added like it should in addition to the Opacity property.
For:
- CheckedPointerOver
- CheckedPressed
- IndeterminatePointerOver
- IndeterminatePressed

Reply:
Can we create an issue for this? I'm not sure what needs to be done here

Reply:
Created https://github.com/unoplatform/Uno.Themes/issues/1257